workers.go

```diff
@@ -194,9 +194,4 @@ func (w *Worker) handleTask(task *queues.Task) {
 
 	elapsed := time.Since(start)
 	log.Info("task complete", "elapsed", elapsed)
-
-	// additionally if any task took longer than 1 minute, log as warning
```

Comment:
unrelated - just noticed we also log long running tasks in a smarter way in `tasks.Perform`